lighthouse-core/test/gather/gatherers/accessibility-test.js

```diff
@@ -19,23 +19,23 @@ describe('Accessibility gatherer', () => {
 
   it('fails if nothing is returned', () => {
     return accessibilityGather.afterPass({
-      driver: {
+      driver: {executionContext: {
```

Comment:
was this formatting on purpose? it's at least inconsistent with some other files, like `password-inputs-with-prevented-paste-test.js`

Reply:
yes and no. the minimal disruption to existing lines/preservation of signal was a nice byproduct of the way my replace was working, so I didn't correct it. But reflowing works for me if you'd prefer.